Science/Science Onboard/IOHandler.cs

```diff
@@ -47,19 +51,34 @@ public void InitializeSystems()
             this.LEDController.Initialize();
             this.AuxSensors.Initialize();
             this.SysSensors.Initialize();
+            this.Music.Initialize();
         }
 
         /// <summary>
         /// Immediately stops all systems.
         /// </summary>
         public void EmergencyStop()
         {
-            //this.RailController.EmergencyStop();
-            this.DrillController.EmergencyStop();
-            //this.SampleController.EmergencyStop();
-            this.LEDController.EmergencyStop();
-            this.AuxSensors.EmergencyStop();
-            this.SysSensors.Initialize();
+            try { this.Music.EmergencyStop(); }
```

Comment:
Might be a good idea to use some OO here and collect the controllers into a subclass, or have them implement some sort of a `EmergencyStoppable` interface.